test/native/setup-after-env.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+import { toBeVisible } from './custom-matchers/to-be-visible';
```

Comment:
```suggestion
import { toBeVisible } from './matchers/to-be-visible';
```

Nitpick: IMO the term "custom" is likely unnecessary. 

N.b. this suggestion should not be accepted directly here, as the file paths need to be updated as well.

Reply:
To be honest, I wasn't sure about whether to add that term. I ended up picking `custom` because I saw references in either the `@testing-library/jest-native` [README](https://github.com/testing-library/jest-native) and in the Jest documentation ([reference](https://jestjs.io/docs/expect#custom-matchers-api)).

In any case, I like the idea of renaming it 👍 .

Reply:
I applied this suggestion in https://github.com/WordPress/gutenberg/pull/38254/commits/b2cd5b411a4e600a0eba3768dd2ca28300733da6.